### PR TITLE
Update broken link in beta-deployment.md

### PR DESCRIPTION
**Pull request by Ian-Phelon**
Corrected the link to the intended directory.

### CODE REVIEW

**Comment by Ian-Phelon** [comment]
@googlebot I signed it!

File: docs/getting-started/android/beta-deployment.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Deploy to Beta distribution services using _fastlane_
 
-If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](setup/#setting-up-supply) before continuing.
+If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](../setup/#setting-up-supply) before continuing.
```
Comment (revolter) [inline docs/getting-started/android/beta-deployment.md:5]: Please change this to
```suggestion
If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](/getting-started/android/setup/#setting-up-supply) before continuing.
```
so it's consistent with the rest of the codebase (we don't use relative URLs at all).

**Review by revolter — CHANGES_REQUESTED** [review]
Great catch 👍🏻

**Review by Ian-Phelon — COMMENTED** [review]
Thanks for the help

**Review by rogerluan — APPROVED** [review]
LGTM 👍 Test failures are unrelated to this PR, I'm fine with merging this in 👌

**Comment by rogerluan** [comment]
Thank you for your contribution @Ian-Phelon ! 🎉

**Comment by rogerluan** [comment]
Ah, actually I can't merge it in because of the failed tests, even though they're unrelated to this PR. 😥 
@Ian-Phelon could you try taking care of them? It's failing because the docs on the page you edited are outdated, referring to an _action_ that doesn't exist anymore (`crashlytics`), because Crashlytics itself (the one from Fabric) kinda doesn't exist anymore 😅 idk exactly if there's an equivalent for this action, or if those sections/mentions could be deleted 🤔 thoughts @revolter ?

**Comment by revolter** [comment]
Oh, I forgot. This only needs a rebase. I fixed the tests in #1103.

**Review by revolter — APPROVED** [review]

**Comment by rogerluan** [comment]
Ahh amazing @revolter thanks!! 🌟

**Comment by Ian-Phelon** [comment]
Thanks @revolter and @rogerluan for helping me to accomplish my first PR.

**Pull request merged**